components/automate-chef-io/content/docs/infra-server.md

```diff
@@ -12,6 +12,12 @@ toc = true
     weight = 60
 +++
 
+{{% warning %}}
+
+Chef Automate does not deploy Chef Manage alongside Chef Infra Server.
```

Comment:
> Chef Automate does not support installing any Chef Infra Server add-ons, such as Chef Manage or Push Jobs Server.

Reply:
do we want to mention that supermarket is not supported as well?

Reply:
rewrite:

Chef Automate will not deploy the following Chef Infra Server add-ons: 
- Chef Manage
- Push Jobs Server
- Supermarket

Reply:
technically supermarket isn't an "addon" as the addons were packages that could be installed with `chef-server-ctl install NAME`. I was happy to confirm this morning that the `chef-server-ctl` command shipped with automate is already well pruned and doesn't have the `install` command at all.

https://docs.chef.io/ctl_chef_server/#install

https://github.com/chef/chef-server/blob/master/src/chef-server-ctl/plugins/install.rb#L4-L8

(reporting is already EOL a while ago).

It's more technically correct that Supermarket cannot integrate with Automate Infra Server because we don't ship the `oc-id` service which is what supermarket uses to authenticate. We haven't ruled out doing something with Automate on-premises and Supermarket but nothing is currently planned either.

Reply:
So maybe two separate warnings. 

```
Chef Automate will not deploy the Chef Infra Server add-ons Chef Manage
and Push Jobs Server.
```
and 
```
Supermarket cannot authenticate users on Chef Infra Server that is deployed 
with Chef Automate.
```

Reply:
sounds good to me!